test/test.js

```diff
@@ -294,8 +285,7 @@ describe('sourcemap-concat', function() {
     });
     builder = new broccoli.Builder(node);
     return builder.build().then(function(result) {
-      expectFile('nothing.js').in(result);
-      expectFile('nothing.map').in(result);
+      expectValidSourcemap('nothing.js').in(result);
```

Comment:
> We could generate a valid one that has nothing in it.

if I understand the situation correctly that is what we are currently doing. we are generating a map for `nothing/*.js` but the whole `nothing` folder doesn't exist, which results in `sources` of the map being empty, which according to the validator is invalid.